Microsoft.Azure.Cosmos/src/Fluent/CosmosClientBuilder.cs

```diff
@@ -424,12 +424,32 @@ public CosmosClientBuilder WithConsistencyLevel(Cosmos.ConsistencyLevel consiste
         }
 
         /// <summary>
-        /// If Open Telemetry listener is subscribed for Azure.Cosmos namespace, There are <see cref="Microsoft.Azure.Cosmos.DistributedTracingOptions"/> you can leverage to control it.<br></br>
+        /// Disables Operation level activity generation even if listener is subscribed to Azure.Cosmos.operation source and clears <see cref="DistributedTracingOptions"/> configured for this client instance.
```

Comment:
For consistency with the other property, source name should be the same?

```suggestion
        /// Disables Operation level activity generation even if listener is subscribed to "Azure.Cosmos.Operation" source.
```

Also, why do we mention that it clears the options? This is the client builder, how does the user set the DistributedTracingOptions and call Disable after?

Reply:
DistributedTracingOptions is internal as of now. No need to mention it is the doc yet.
Right now, this feature in on by default and builder has API to disable it.